main.js

```diff
@@ -161,11 +161,38 @@ const mail = document.getElementById('mail');
 const errorMessage = document.querySelector('.error-message');
 
 formdata.addEventListener('submit', (e) => {
-  const RegExp = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
-  if (mail.value === mail.value.match(RegExp)) {
+  const Regex = /^[a-z0-9.!#$%&'*+/=?^_`{|}~-]+@[a-z0-9-]+(?:\.[a-z0-9-]+)*$/;
+  if (mail.value.match(Regex)) {
     errorMessage.textContent = '';
   } else {
     e.preventDefault();
     errorMessage.innerHTML = '** FORM NOT SENT <br> ** Invalid Email Address ';
   }
 });
+
```

Comment:
Please make sure that entered values are saved in the browser after filling the form, and not submitting it.